AutoGear.lua

```diff
@@ -2224,6 +2224,7 @@ AutoGearFrame:RegisterEvent("GOSSIP_CONFIRM_CANCEL")        --Fires when an atte
 AutoGearFrame:RegisterEvent("GOSSIP_ENTER_CODE")            --Fires when the player attempts a gossip choice which requires entering a code
 AutoGearFrame:RegisterEvent("GOSSIP_SHOW")                  --Fires when an NPC gossip interaction begins
 AutoGearFrame:RegisterEvent("UNIT_QUEST_LOG_CHANGED")       --Fires when a unit's quests change (accepted/objective progress/abandoned/completed)
+AutoGearFrame:RegisterEvent("BAG_UPDATE_DELAYED")			--Fires when any bag contents are updated, including moving items. Fires on delay which avoided nil issues with BAG_UPDATED event
```

Comment:
We do not want the behavior to which the registration of this event leads.  It would prevent removing an item manually.  AutoGear would fight against the player immediately instead of allowing the player to move their items manually as they would wish.  AutoGear should only automatically equip an item under these circumstances:

- the player's character receives a gear item for the first time from outside their inventory, like from a loot roll, vendor, or a quest reward
- the player's character's gear evaluation situation changes, like when changing specialization
- the player has manually initiated a scan

Reply:
See previous comment, I changed this locally to listen to BAG_NEW_ITEMS_UPDATED

Reply:
OK.  My other comment touches on the most likely reason the auto-equipping "stopped working": the chat filter no longer matching.  You may want to start there, and/or make use of `/eventtrace` to find exactly what event is thrown with which arguments when receiving new gear from outside the player's inventory.